wpilibc/src/main/native/include/IterativeBase.h

```diff
@@ -0,0 +1,48 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+
+/**
```

Comment:
Does Doxygen actually generate the docs for this properly if it's not next to the class definition? We generally put the class comment inside the namespace.